packages/nimble-components/src/wafer-map/modules/rendering.ts

```diff
@@ -10,13 +10,28 @@ export class RenderingModule {
 
     public constructor(private readonly wafermap: WaferMap) {}
 
-    public drawWafer(): void {
+    public updateSortedDies(): void {
+        this.dies = this.wafermap.dataManager.diesRenderInfo.sort((a, b) => {
+            if (a.fillStyle > b.fillStyle) {
+                return 1;
+            }
+            if (b.fillStyle > a.fillStyle) {
+                return -1;
+            }
+
+            return 0;
+        });
+        this.updateDrawnWafer();
+    }
+
+    public updateDrawnWafer(): void {

```

Comment:
nit: I think the name change for this function seems unnecessary/overly verbose. The idea that the wafer has already been "drawn" seems irrelevant, and generally the term "update" doesn't always mean a complete refresh happens, which is happening in the implementation. So, `drawWafer` still seems entirely appropriate for the method name.